src/components/Example/Example.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getMembers } from "../../functions/members";
+
+const Example = () => {
+  const [members, setMembers] = useState([]);
+
+  useEffect(() => {
+    const fetchMembers = async () => {
+      const members = await getMembers();
+      console.log(members);
+      setMembers(members);
+    }
+
+    fetchMembers();
+  }, []);
+
+  return (
+    <div>
+      <h2>Members</h2>
+      <ul>
+        {members.map(member => (
+          <li style={{ listStyle: 'none' }} key={member.id}>
```

Comment:
I'm getting a warning about a key prop not being assigned (though I know this probably just a placeholder for now anyway). Looking at the data, I don't think each member actually has an id value. Maybe you meant `member.userID`?